spec

```diff
@@ -69,7 +57,6 @@ beginwave
 	name "game"
 	include "code"
 	include "static"
-	include "texture"

```

Comment:
There are references to `_textureSegmentRomStart` and `_textureSegmentRomEnd` in boot.c that need to be removed too